quarkus-azure-services.tf

```diff
@@ -32,3 +32,16 @@ resource "github_team_membership" "quarkus_azure_services" {
   username = each.value
   role     = "maintainer"
 }
+
+# Create CI environment secrets
+resource "github_actions_environment_secret" "quarkus_azure_services" {
+  for_each = {
+    "AZURE_CLIENT_ID"       = "foo"
+    "AZURE_TENANT_ID"       = "bar"
+    "AZURE_SUBSCRIPTION_ID" = "blah"
+  }
+  repository      = github_repository.quarkus_azure_services.name
+  environment     = "ci"
+  secret_name     = each.key
+  plaintext_value = each.value
```

Comment:
Do you need them to be encrypted? Just change to `encrypted_value` and generate teh encrypted value using https://docs.github.com/en/rest/actions/secrets?apiVersion=2022-11-28#create-or-update-an-environment-secret (I can come up with a script for that if necessary)

Reply:
Thank you @gastaldi. They needn't be encrypted as there are no **secrets** for these values. I just pushed a commit to update them with valid values.